### ISSUE TITLE
"Fix" the damage dealt by guns

### ISSUE THREAD

**Issue by Kirby859** [FlansMods/FlansMod#110]
Alright, let's say I have two guns. One is called the aks-9001um. It will have stats of:
damage 5
accuracy 0
shootdelay 1
(full auto)

And my second gun would be called the ak-9001m. It will have stats of:
damage 7
accuracy 0
shootdelay 2
(full auto)

Now, if we calculate the DPS of each gun, the aks-9001um would have a DPS of 100 half-hearts of damage per second, and the ak-9001m would have a DPS of 70 half-hearts of damage per second. So clearly, the aks-9001um is a better choice (assuming all other stats of the two guns are the same), right?

haha no

Minecraft has somewhat of a balancing system, where whenever you take damage, you become invulnerable for a short period of time. This prevents things like modded controllers/mice/etc. from dominating the PVP world. The problem is that this takes place in Flan's Mod when I think it shouldn't. Any two people should be able to hold down the right mouse button until a person dies, so this "balancing" feature should somehow be bypassed in Flan's Mod (at least, with guns).

With the "balancing" in place, fast-rpm/low-damage guns are useless. THE best guns are the guns that have one of the following characteristics:
1. one-shot kill
2. high damage, low-medium rpm
Because of the "balancing," all guns that have an RPM above a certain number don't get to use their high RPM because a player cannot receive damage that fast.

tl;dr let players receive full damage from high-rpm guns by somehow bypassing Minecraft's damage invulnerability
